app/config/question-bank.js

```diff
@@ -2372,19 +2375,19 @@ const questionBank = {
             prefixSufix: [
               {
                 linkedPrefix: 'Increase: ',
-                linkedSufix: 'm³',
+                linkedSufix: 'm³'
               },
               {
                 linkedPrefix: 'Size: ',
-                linkedSufix: 'm²',
+                linkedSufix: 'm²'
               },
               {
                 linkedPrefix: 'Size: ',
-                linkedSufix: 'm²',
-              },
+                linkedSufix: 'm²'
+              }
             ],
             linkedQuestionyarkey: ['serviceCapacityIncrease', 'coverSize', 'existingCoverSize'],
-            dependentQuestionKeys: ['storageType', 'coverType', 'existingCoverType', 'separatorOptions'],
+            dependentQuestionKeys: ['storageType', 'coverType', 'existingCoverType', 'separatorType']
```

Comment:
leave this as separatorOptions pls. I need to add all the selected separator details to a unique yarkey for the sidebars/project summary/email/dora so please leave this as is

Reply:
its a bug raised by Rev it was there just showing the wrong heading

Reply:
If its a header issue, that should be sorted in models.js (See line 33-50)

Reply:
I found and fixed the issue (moels.js switch case didn't have a break for separator. Its fixed now with separatorOptions